FirebaseSessions/Tests/Unit/SessionStartEventTests.swift

```diff
@@ -113,4 +113,24 @@ class SessionStartEventTests: XCTestCase {
       XCTAssertEqual(event.proto.application_info.apple_app_info.os_name, expected)
     }
   }
+
+  func test_convertLogEnvironment_convertsCorrectly() {
+    let expectations: [(given: String, expected: firebase_appquality_sessions_LogEnvironment)] = [
+      ("prod", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_PROD),
+      ("staging", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_STAGING),
+      ("autopush", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_AUTOPUSH),
+      ("PROD", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_PROD),
+      ("STAGING", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_STAGING),
+      ("AUTOPUSH", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_AUTOPUSH),
+      ("random", firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_UNKNOWN),
+    ]
+
+    expectations.forEach { (given: String, expected: firebase_appquality_sessions_LogEnvironment) in
+      appInfo.environment = given
```

Comment:
For this, I worry we're not testing the `environment` getter itself. We can instead set `ProcessInfo.processInfo.environment` and leave off the mock for appInfo.environment.

Alternatively, we can add a test to ApplicationInfoTest and do the `ProcessInfo.processInfo.environment` testing there